spec/system/prior_authority/additional_costs_spec.rb

```diff
@@ -61,12 +61,24 @@
           click_on 'Delete'
           expect(page).to have_content 'Are you sure you want to delete this additional cost?'
           click_on 'Yes, delete it'
-          expect(page).to have_content 'Do you want to add another additional cost?'
-          choose 'No'
-          click_on 'Save and continue'
+          expect(page).to have_content 'You have deleted the additional cost'
+          expect(page).to have_content 'Primary quote summary'
           expect(page).to have_content 'No additional costs added'
         end
 
+        context 'when I have added multiple costs' do
+          before do
+            create :additional_cost, prior_authority_application: PriorAuthorityApplication.first
+          end
+
+          it 'removes the cost then takes me back to the cost list' do
+            click_on 'Delete'
+            click_on 'Yes, delete it'
+            expect(page).to have_content 'You have deleted the additional cost'
```

Comment:
It's not relevant to this PR but I have noticed that we dont' really assert the path of the page much.

This might help times when we refactor a path (like the recent turboframe stuff) and the tests pass but things still break. Is there a good reason for this that I'm not aware of? 😄